podcasts/TranscriptViewController.swift

```diff
@@ -301,32 +359,38 @@ class TranscriptViewController: PlayerItemViewController {
     }
 
     private func updateTextMargins() {
-        let margin = self.view.readableContentGuide.layoutFrame.minX + 8
+        let margin = self.view.readableContentGuide.layoutFrame.minX + Sizes.textMargin
         transcriptView.textContainerInset = .init(top: 0.75 * Sizes.topGradientHeight, left: margin, bottom: bottomContainerInset, right: margin)
     }
 
     @MainActor
     private func refreshText() {
+        if let errorText = errorLabel.text {
+            errorLabel.attributedText = NSAttributedString(string: errorText, attributes: makeStyle())
+        }
+
         guard let transcript else {
             return
         }
         transcriptView.attributedText = styleText(transcript: transcript)
     }
 
     private func show(transcript: TranscriptModel) {
-            activityIndicatorView.stopAnimating()
-            self.previousRange = nil
-            self.transcript = transcript
-            transcriptView.attributedText = styleText(transcript: transcript)
+        searchButton.isHidden = false
+        errorView.isHidden = true
+        activityIndicatorView.stopAnimating()
+        self.previousRange = nil
```

Comment:
Same about the previous comment, I believe we can move that to its own method.

Reply:
Done here: https://github.com/Automattic/pocket-casts-ios/pull/2007/commits/f8228ca3732aee5296bf0e7c85228b49f07fa5fd